soak/run_soak_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+PYTEST_LOG_LEVEL=$(echo "${PYTEST_LOG_LEVEL:-"info"}" | tr '[:upper:]' '[:lower:]')
+PYTEST_NUM_THREADS=${PYTEST_NUM_THREADS:-"auto"}
+SOAK_TEST_DURATION_MINUTES="${SOAK_TEST_DURATION_MINUTES:-90}"
+
+USAGE="
+Usage:
+  $(basename "$0")
+
+Environment variables:
+  START_TIME_EPOCH_SECONDS:   Start time of soak test run in seconds from epoch.
+  SOAK_TEST_DURATION_MINUTES: Number of minutes to run soak test. Default: 90 minutes
+  CONTROLLER_E2E_PATH:        Path where e2e tests reside for ack service controller.
+  PYTEST_LOG_LEVEL:           Set to any Python logging level for the Python tests.
+                              Default: info
+"
+
+echo "[INFO] Starting the soak test run."
+echo "[INFO] START_TIME_EPOCH_SECONDS is $START_TIME_EPOCH_SECONDS"
+((END_TIME_EPOCH_SECONDS= "$START_TIME_EPOCH_SECONDS" + "$SOAK_TEST_DURATION_MINUTES"*60))
+echo "[INFO] END_TIME_EPOCH_SECONDS is $END_TIME_EPOCH_SECONDS"
+
+pushd "${CONTROLLER_E2E_PATH}" 1>/dev/null
+# Treat e2e directory as a module to run bootstrap as scripts
+export PYTHONPATH=..
+pip install -r requirements.txt
```

Comment:
This is already being done in the Dockerfile?

Reply:
This is a different requirements.txt

The one in Dockerfile is from test-infra repository. This one is from service-controller test directory.

Reply:
You don't need to run both. The one in the service-controller pulls a specific version of the `test-infra` as part of its dependency tree. See SageMaker's for example: https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/test/e2e/requirements.txt